Tests/SonarScanner.MSBuild.PreProcessor.Test/ProcessedArgsTests.cs

```diff
@@ -303,21 +303,27 @@ public void ProcArgs_HostUrl_SonarcloudUrl_HostUrlAndSonarcloudUrlMissing()
         sut.IsValid.Should().BeTrue();
     }
 
-    [TestMethod]
-    public void ProcArgs_HostUrl_SonarcloudUrl_HostUrlIsDefaultSonarcloud()
+    [DataTestMethod]
+    [DataRow("https://sonarcloud.io")]
+    [DataRow("https://SonarCloud.io")]
+    [DataRow("https://SONARCLOUD.IO")]
+    [DataRow("https://sonarqube.us")]
+    [DataRow("https://SonarQube.us")]
+    [DataRow("https://SONARQUBE.US")]
```

Comment:
`url = url.TrimEnd('/');` in `IsKnownUrl` should be under test:
```suggestion
    [DataRow("https://SONARCLOUD.IO")]
    [DataRow("https://sonarcloud.io/")]
    [DataRow("https://sonarqube.us")]
    [DataRow("https://SonarQube.us")]
    [DataRow("https://SONARQUBE.US")]
    [DataRow("https://sonarqube.us/")]
```